### ISSUE TITLE
Remote SSH requires SFTP and there is no single word mentioning SFTP in the article

### ISSUE THREAD

**Issue by shoter** [MicrosoftDocs/visualstudio-docs#6962]
I was unable to connect remotely to our docker containers on linux and nothing was working for long time until one of our engineers discovered that Visual Studio is trying to use SFTP protocol.

We could connect remotely after installing SFTP. Below command was good enough:
```
dpkg -i /netsetup/openssh-sftp-server_8.2p1-r0_amd64.deb
```

Personally it is super hard to work with Microsoft technologies if you have logs that are not telling what is going on. Only log we got on this operation was `Channel was closed` or something super similar. It's bizarre that this requirement is not documented :\

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b816d9aa-344b-6956-11e9-2f58d24f604c
* Version Independent ID: 6d2000ec-5a1c-c824-554c-ff5fe4c806e6
* Content: [Attach to a process running on a Docker container - Visual Studio (Windows)](https://docs.microsoft.com/en-us/visualstudio/debugger/attach-to-process-running-in-docker-container?view=vs-2019)
* Content Source: [docs/debugger/attach-to-process-running-in-docker-container.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/debugger/attach-to-process-running-in-docker-container.md)
* Product: **visual-studio-windows**
* Technology: **vs-ide-debug**
* GitHub Login: @Mikejo5000
* Microsoft Alias: **mikejo**

**Comment by Mikejo5000** [comment]
@shoter Thanks for letting us know. I'll ask about this. Do you know whether you had the .NET Cross-platform Development workload installed?

**Comment by shoter** [comment]
Yes, I had it installed:
![image](https://user-images.githubusercontent.com/3819892/129245970-be7eb040-4fcc-4dc5-a7d6-88ba20eaa946.png)
As I said everything was working after we installed SFTP on the device we were debugging.
I did not need to install anything on my computer.

Also our devices have fectory reset every restart and every restart I need to install SFTP. So it is not a fluke that SFTP is required. I installed it maaaaaaaany times.
